### PR TITLE
RD-1072 Fix misalignment when displaying parameter values

### PR DESCRIPTION
**Pull request by qooban**
Fixed misalignment coming from parameter values, e.g. in this case:

https://user-images.githubusercontent.com/5202105/121895224-437f5d00-cd20-11eb-93f4-6c8e6cbfa760.mp4

See comments about additional small fixes.

### CODE REVIEW

File: widgets/common/src/ParameterValue.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 // @ts-nocheck File not migrated fully to TS
 import { types } from 'cloudify-ui-common';
+import highlighterStyles from 'react-syntax-highlighter/dist/esm/styles/hljs/idea';
```
Comment (qooban) [inline widgets/common/src/ParameterValue.tsx:3]: I discovered another bug, so fixed it right away.

It was necessary to import the styles as the classes used previously (`hljs-keyword`, `hljs-number`) are no longer present.

File: widgets/common/src/ParameterValue.tsx [hunk 2]
```diff
@@ -14,47 +15,33 @@ export default class ParameterValue extends React.Component {
         const { HighlightText } = Stage.Basic;
         const { Url } = Stage.Utils;
 
-        const commonStyle = { padding: '0.5em', whiteSpace: 'pre-wrap', wordBreak: 'break-word' };
+        const commonStyle = {
+            display: 'inline-block',
```
Comment (qooban) [inline widgets/common/src/ParameterValue.tsx:19]: This is the actual fix for RD-1072.

File: widgets/common/src/ParameterValueDescription.tsx [hunk 3]
```diff
@@ -11,7 +11,7 @@ export default function ParameterValueDescription() {
             flowing
             header="Value style"
             content={
-                <List bulleted>
+                <List>
```
Comment (qooban) [inline widgets/common/src/ParameterValueDescription.tsx:14]: After changing display method in `ParameterValue` the bullets were not displayed properly, so I decided to just remove the bullet and do not spend more time on making special alignments.

**Review by kubama — APPROVED** [review]

**Review by Gelio — APPROVED** [review]

**Comment by qooban** [comment]
Thanks for the reviews. I merged the PR. As it was just small style change, I did not run system tests.

**Pull request merged**